msticpy/nbtools/timeline.py

```diff
@@ -320,18 +320,15 @@ def display_timeline_values(
             p_series = []
             # create default plot args
             plot_args: Dict[str, Any] = dict(
-                x=time_column,
-                alpha=0.7,
-                source=row_source,
-                legend_label=str(inline_legend),
+                x=time_column, alpha=0.7, source=row_source, legend_group=legend_column
             )
             if "vbar" in plot_kinds:
-                p_series.append(plot.vbar(top=y, width=4, color="color", **plot_args))
+                p_series.append(plot.vbar(top=y, width=4, color=color, **plot_args))
```

Comment:
Doesn't this set all series to a single color? The "color" parameter here is meant to reference the value set in the call to _create_data_grouping which is mapped for each group. Maybe we should re-think this - how a user can specify the color map to use or something.

Reply:
Yep - changed.